scopes/compilation/compiler/compiler.cmd.ts

```diff
@@ -38,7 +39,7 @@ export class CompileCmd implements Command {
 
     let outputString = '';
     compilerOptions.deleteDistDir = true;
-    await this.compile.compileComponents(components, compilerOptions);
+    await this.compile.compileComponents(components, { ...compilerOptions, origin: CompileOrigin.CmdReport });
```

Comment:
maybe move the `deleteDistDir` from one line above into this as well. so we won't have diff between `deleteDistDir` and `origin`
(same in the `json` func)

Reply:
not sure also about the name origin, maybe initiator? maybe requestor? or requestOrigian?

Reply:
Changed the name to `CompilationInitiator`